From 315cb6c6785dfb5f2275d87cadca5a09ad8ec36c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=8D=C3=B1igo=20Huguet?= <ihuguet@redhat.com>
Date: Fri, 21 Jan 2022 08:49:01 +0100
Subject: [PATCH 02/36] rtw89: Fix two spelling mistakes in debug messages
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Bugzilla: http://bugzilla.redhat.com/2033291

commit c51ed74093d45af2a25af7f26dd8d0532f7aec3e
Author: Colin Ian King <colin.king@canonical.com>
Date:   Fri Oct 15 11:50:04 2021 +0100

    rtw89: Fix two spelling mistakes in debug messages

    There are two spelling mistakes in rtw89_debug messages. Fix them.

    Signed-off-by: Colin Ian King <colin.king@canonical.com>
    Acked-by: Ping-Ke Shih <pkshih@realtek.com>
    Signed-off-by: Kalle Valo <kvalo@codeaurora.org>
    Link: https://lore.kernel.org/r/20211015105004.11817-1-colin.king@canonical.com

Signed-off-by: Íñigo Huguet <ihuguet@redhat.com>
---
 drivers/net/wireless/realtek/rtw89/phy.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/realtek/rtw89/phy.c b/drivers/net/wireless/realtek/rtw89/phy.c
index 53c36cc82c57..ab134856baac 100644
--- a/drivers/net/wireless/realtek/rtw89/phy.c
+++ b/drivers/net/wireless/realtek/rtw89/phy.c
@@ -1715,7 +1715,7 @@ static s32 rtw89_phy_multi_sta_cfo_calc(struct rtw89_dev *rtwdev)
 			target_cfo = clamp(cfo_avg, max_cfo_lb, min_cfo_ub);
 		} else {
 			rtw89_debug(rtwdev, RTW89_DBG_CFO,
-				    "No intersection of cfo torlence windows\n");
+				    "No intersection of cfo tolerance windows\n");
 			target_cfo = phy_div(cfo_khz_all, (s32)sta_cnt);
 		}
 		for (i = 0; i < CFO_TRACK_MAX_USER; i++)
@@ -2749,7 +2749,7 @@ static void rtw89_phy_dig_dyn_pd_th(struct rtw89_dev *rtwdev, u8 rssi,
 			    dig->igi_rssi, final_rssi, under_region, val);
 	} else {
 		rtw89_debug(rtwdev, RTW89_DBG_DIG,
-			    "Dynamic PD th dsiabled, Set PD_low_bd=0\n");
+			    "Dynamic PD th disabled, Set PD_low_bd=0\n");
 	}
 
 	rtw89_phy_write32_mask(rtwdev, R_SEG0R_PD, B_SEG0R_PD_LOWER_BOUND_MSK,
-- 
2.13.6

